packages/ui-components/src/components/absolute-time-picker/absolute-time-picker.config.ts

```diff
@@ -6,7 +8,7 @@ export const DATE_INPUT_MASK = 'DD-MM-YYYY';
 export const CALENDAR_MASK = 'YYYY-MM-DD';
 export const CALENDAR_DATE_TIME_MASK = 'YYYY-MM-DD HH:mm:ss';
 export const CALENDAR_INPUT_MIN_DATE = '01-01-2018 00:00:00';
-export const CALENDAR_INPUT_MAX_DATE = '31-12-2050 23:59:59';
-export const MAX_INPUT_YEAR = 2050;
+export const CALENDAR_INPUT_MAX_DATE = dayjs('31-12-2050 23:59:59', DATETIME_INPUT_MASK).add(50, 'year').format(DATETIME_INPUT_MASK);
```

Comment:
Should these configs be removed in the future? If we don't provide a min or max on the calendar we should render until infinite.

Reply:
Actually we dont need the date '31-12-2050 23:59:59', we could use today and add 50 years. But I guess i can ignore this limits and use a prop instead.

Reply:
I can remove this configs and use the props instead.